src/android/java/io/jxcore/node/ConnectivityMonitor.java

```diff
@@ -8,43 +8,52 @@
 import android.net.ConnectivityManager;
```

Comment:
The implementation was requested a while back, and thus is implemented. Supposing that its not really used. Also the life cycle monitoring code is supposedly currently not used at all. If indeed we decide that these are not needed, we could then remove them from the plugin.